src/main/kotlin/fi/elsapalvelu/elsa/service/mapper/UserMapper.kt

```diff
@@ -35,6 +35,8 @@ class UserMapper {
                     firstName = userDTO.firstName,
                     lastName = userDTO.lastName,
                     email = userDTO.email,
+                    phoneNumber = userDTO.phoneNumber,
+                    avatar = userDTO.avatar,
```

Comment:
Voisikohan tämänkin mapperin tehdä mapstructilla?

Reply:
Voisi tehdä. JHipster kertoo seuraavaa asiasta: Normal mappers are generated using MapStruct, this one is hand-coded as MapStruct support is still in beta, and requires a manual step with an IDE.